modules/terragrunt/cmd.go

```diff
@@ -0,0 +1,71 @@
+package terragrunt
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/gruntwork-io/terratest/modules/retry"
+	"github.com/gruntwork-io/terratest/modules/shell"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/gruntwork-io/terratest/modules/testing"
+)
+
+func runTerragruntStackCommandE(t testing.TestingT, opts *Options, additionalArgs ...string) (string, error) {
+	args := []string{"stack", "run"}
+	{
+		// check if we are using older version of terragrunt
+		cmd := shell.Command{Command: opts.TerraformBinary, Args: []string{"-experiment", "stack"}}
+		if err := shell.RunCommandE(t, cmd); err == nil {
+			args = prepend(args, "-experiment", "stack")
```

Comment:
I think we don't need this check, since the experiment stack is already completed

Reply:
Will address this in a separate PR and merge this in for now! Thanks!